FUNDABLES.md

```diff
@@ -260,6 +260,17 @@ need funding for user experience work, development, testing,
 infrastructure, potentially platform services (e.g., SMS), and community
 outreach.
 
+### Productionize Malware Detection
+PSF did an [RfP](https://github.com/python/request-for/blob/master/2019-Q4-PyPI/RFP.md) for some security work on [Warehouse](https://github.com/pypa/warehouse) in late 2019, but the goals for [Milestone 2](https://github.com/python/request-for/blob/master/2019-Q4-PyPI/RFP.md#milestone-2---systems-for-automated-detection-of-malicious-uploads) were more ambitious than funding allowed for. The malware detection system is currently in limbo: an interesting prototype with limited practical impact because of the astounding number of false-positives. Here is a sampling of some of the open issues:
+- [Make Malware Verdicts Auditable](https://github.com/pypa/warehouse/issues/7421) - verdicts are removed once the associated project/package is removed, which makes it impossible to evaluate the efficacy of this system
+- [Detect packages being published with typo'ish names](https://github.com/pypa/warehouse/issues/4998) - No progress was made on a typosquatting check.
```

Comment:
Thanks for writing this up @xmunoz!

Since most of the list concentrates on what still needs doing, not on past lapses, I suggest some rephrasings, such as:

```suggestion
- [Detect packages being published with typo'ish names](https://github.com/pypa/warehouse/issues/4998) - Add a typosquatting check.
```